object/types.proto

```diff
@@ -47,6 +47,18 @@ enum MatchType {
 
   // String prefix match
   COMMON_PREFIX = 4;
+
+  // Numerical 'greater than'
+  NUM_GT = 5;
+
+  // Numerical 'greater or equal than'
+  NUM_GE = 6;
+
+  // Numerical 'less than'
+  NUM_LT = 7;
+
+  // Numerical 'less or equal than'
+  NUM_LE = 8;
```

Comment:
`NUM_EQUAL`?

> the previously added STRING_EQUAL / STRING_NOT_EQUAL matchers are suitable for numerical == / !=.

While it is true for now, it looks strange to me. Users should use `STRING_EQUAL` for numbers in their interfaces. Also, it is possible not to have strings-only values in the future.

Reply:
new values will be added when the existing ones are no longer sufficient

Reply:
I had this question as well, but as we're not supporting different notations ("1e8", for example) now, current set is sufficient for the purpose.

Reply:
my main point, i would say in the naming of this message: `MatchType` (it is not bound to `Filter`), no info about what it relates to. i can imagine reusing this for a number representation of smth, then we will have to add `NUM_EQUAL` but we will already have had some users that will be using `STRING_EQUAL`, strange

Reply:
> this enum is not intended to be reused

but there are no words about what it is used for